roles/remediate/tasks/leapp_cgroups-v1_enabled.yml

```
@@ -0,0 +1,68 @@
+---
+- name: leapp_cgroups-v1_enabled | Disable cgroups-v1 in favor of the new version cgroups-v2
+  vars:
+    leapp_report_location: /var/log/leapp/leapp-report.json
+    entry_title: cgroups-v1 enabled on the system
```

```suggestion
    leapp_entry_title: cgroups-v1 enabled on the system
```

let's prefix our vars with leapp to avoid conflicts with users vars